### PR TITLE
Package errpy.0.0.7

### PR DESCRIPTION
**Pull request by jasontatton**
### `errpy.0.0.7`
Errpy: An Error Recovering Rust Python Parser
This is a new project to provide an error-recovering Python parser, implemented in Rust based on tree-sitter. Our short-term goal is to use it in the pyre type checker, in order to provide better IDE features.

---
* Homepage: https://github.com/facebook/errpy
* Bug tracker: https://github.com/facebook/errpy/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

**Comment by jasontatton** [comment]
Looks like the test failures here are only in relation to some older Linux distributions which have an older version of rust installed?

File: packages/errpy/errpy.0.0.7/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Errpy: An Error Recovering Rust Python Parser"
+description:
+  "This is a new project to provide an error-recovering Python parser, implemented in Rust based on tree-sitter. Our short-term goal is to use it in the pyre type checker, in order to provide better IDE features."
+maintainer: ["pyre@meta.com"]
+authors: ["pyre@meta.com"]
+license: "MIT"
```
Comment (kit-ty-kate) [inline packages/errpy/errpy.0.0.7/opam:8]:
```suggestion
license: "MIT"
dev-repo: "git+https://github.com/facebook/errpy"
```

**Comment by kit-ty-kate** [comment]
The `dev-repo` field was missing but otherwise good to go once CI finishes.

**Comment by jasontatton** [comment]
Awesome, thanks so much @kit-ty-kate . I will get the `dev-repo` field added for future publishes

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**